lib/msf/ui/console/command_dispatcher/developer.rb

```diff
@@ -512,20 +515,35 @@ def cmd_time_help
     print_line
   end
 
-  private
+  protected
+
+  def source_directories
```

Comment:
Extracting this out so multiple source directories can be reloaded i.e. for Pro, and potentially some custom user extensions